.vscode/settings.json

```diff
@@ -1,3 +1,18 @@
 {
```

Comment:
All the settings here looks great to me, however they're opinionated and since I don't work on the docs, I don't really know. If they're what people want, maybe @delucis would have a better idea!

Reply:
Thanks! Will ask Chris to weigh in here!

Reply:
Hi @Afrowave - Discussed this with Chris and we think these settings are too opinionated for putting as official documentation. We are happy to add the Sublime link, though, so if you can amend this PR so that the only change is the link that would be wonderful!

Reminder, any future PRs should be marked as "maintainers may edit" so that we can just fix little things like this ourselves and not bother you.  🙌